fish-be/src/main/game/card.ts

```diff
@@ -25,4 +25,13 @@ export class Card implements ICard {
 
     return suits && values;
   }
+
+  isSameHalfSuit(card: Card): boolean {
+    const suits = this.suit === card.suit;
+    const thisHalf = Math.floor(this.value / 6);
+    const cardHalf = Math.floor(card.value / 6);
+    const values = thisHalf === cardHalf;
```

Comment:
What's the magic number 6 for? Could you provide more explanation to why the magic number is used? Could we have a constant for it instead?